net/netfilter/nf_tables_api.c

```diff
@@ -7955,14 +7958,31 @@ static void nft_set_commit_update(struct list_head *set_update_list)
 	}
 }
 
+static unsigned int nft_gc_seq_begin(struct nftables_pernet *nft_net)
+{
+        unsigned int gc_seq;
+
+        /* Bump gc counter, it becomes odd, this is the busy mark. */
+        gc_seq = READ_ONCE(nft_net->gc_seq);
+        WRITE_ONCE(nft_net->gc_seq, ++gc_seq);
+
+        return gc_seq;
+}
+
+static void nft_gc_seq_end(struct nftables_pernet *nft_net, unsigned int gc_seq)
+{
+        WRITE_ONCE(nft_net->gc_seq, ++gc_seq);
+}
+
 static int nf_tables_commit(struct net *net, struct sk_buff *skb)
 {
+	struct nftables_pernet *nft_net = nft_pernet(net);
 	struct nft_trans *trans, *next;
+	unsigned int base_seq, gc_seq;
```

Comment:
This should be a part of a later commit:
netfilter: nf_tables: fix GC transaction races with netns and netlink event exit path
https://github.com/ctrliq/kernel-src-tree/commit/6a33d8b73dfac0a41f3877894b38082bd0c9a5bc#diff-233a9dea2c513f5d208bba55e0f8ef4e8a29f5c584d30b5815cfa6c2cf5361ecR9742